### ISSUE TITLE
crash NRO & OVL switch firmware 16.0.2

### ISSUE THREAD

**Issue by Fujin33** [WerWolv/EdiZon#190]
The homebrew application crashes in 2168-0002 when your update the backup. It also crashes in the tesla menu in 2001-0123 at the start. I reinstalled all my homebrew with the versions found on May 7 at 3:00 p.m. twice in a row. nothing works. Everything else works.

Updated yesterday: ATMOSPHERE 1.5.2 | 6.0.3 | FIRMWARE 16.0.2

Today: Edizon 3.1.0 | Tesla Menu 1.2.2 | Nx-ovlloader 1.0.7 | sysmodules 1.3.1 | emuiibo 1.0.0.

Julien

**Comment by Simsiano** [comment]
Having the exact identical problem...

**Comment by tibos93** [comment]
Je confirme cheat en erreur sur la version 16.0.1

**Comment by tibos93** [comment]
error code 2001-0123(0xf601)

**Comment by tibos93** [comment]
Allez vérifier dans votre carte SD dans : atmosphere\fatal_reports

Et vérifiez le dernier rapport dans "Program ID : XXXXXXXXXXXXXXXX"

Le mien était "420000000007e51a"

Ensuite, allez dans : "atmosphere\contents"
Et supprimez le dossier qui porte le même nom a le L'ID du programme vous est montré et normalement tout est fait

**Comment by Simsiano** [comment]
> Allez vérifier dans votre carte SD dans : atmosphere\fatal_reports
> 
> Et vérifiez le dernier rapport dans "Program ID : XXXXXXXXXXXXXXXX"
> 
> Le mien était "420000000007e51a"
> 
> Ensuite, allez dans : "atmosphere\contents" Et supprimez le dossier qui porte le même nom a le L'ID du programme vous est montré et normalement tout est fait

It's the same Program ID for me but after I delete it it just reappear.

**Comment by tibos93** [comment]
Pour le moment impossible d'utilisé les cheats code sur 16.0.0

**Comment by Fujin33** [comment]
Same for me, it doesn't change anything. I just noticed that I also no longer have the option to disable applet mode even by removing the ";" in the config, if anyone has info where to report the problem.

**Comment by tibos93** [comment]
rollback firmware 14.0.1 work now

**Comment by Fujin33** [comment]
Yes it is a possibility but certainly not the solution. @WerWolv will find the solution for sure.

**Comment by Simsiano** [comment]
If anyone interested by the problem...the author says he stopped working on this project, the last firmware EdiZon will work is 14.0.
@Fujin33
This is where @WerWolv said it: https://github.com/WerWolv/EdiZon/issues/185#issuecomment-1277716144
If anywant want EdiZon for fw 16 look at this repo: https://github.com/proferabg/EdiZon-Overlay

**Comment by Fujin33** [comment]
@Simsiano Thanks, that's a good answer.
